features/aanhef.feature

```diff
@@ -583,46 +583,46 @@ Rule: Voor het bepalen van de aanhef gaat gebruik van de adellijke titel van de
-      | GI                      | B                               | N           | Hoogwelgeboren vrouwe   |
-      | BS                      | P                               | E           | Hoogwelgeboren vrouwe   |
-      | BS                      | P                               | P           | Hoogheid                |
-      | BS                      | P                               | V           | Hoogheid                |
-      | BS                      | P                               | N           | Hoogheid                |
-      | GI                      | R                               | E           | Hooggeboren vrouwe      |
-      | GI                      | R                               | P           | Geachte mevrouw De Boer |
-      | GI                      | R                               | V           | Hooggeboren vrouwe      |
-      | GI                      | R                               | P           | Hooggeboren vrouwe      |
-      | PS                      | B                               | E           | Hoogheid                |
-      | PS                      | B                               | P           | Hoogwelgeboren vrouwe   |
-      | PS                      | B                               | V           | Hoogwelgeboren vrouwe   |
-      | PS                      | B                               | N           | Hoogwelgeboren vrouwe   |
+      | adellijkeTitelPredicaat | adellijkeTitelPredicaat partner | naamgebruik | aanhef                                |
+      | GI                      | B                               | E           | Hooggeboren vrouwe                    |
+      | GI                      | B                               | P           | Hoogwelgeboren vrouwe                 |
+      | GI                      | B                               | V           | Hoogwelgeboren vrouwe                 |
+      | GI                      | B                               | N           | Hoogwelgeboren vrouwe                 |
+      | BS                      | P                               | E           | Hoogwelgeboren vrouwe                 |
+      | BS                      | P                               | P           | Hoogheid                              |
+      | BS                      | P                               | V           | Hoogheid                              |
+      | BS                      | P                               | N           | Hoogheid                              |
+      | GI                      | R                               | E           | Hooggeboren vrouwe                    |
+      | GI                      | R                               | P           | Geachte mevrouw De Boer               |
+      | GI                      | R                               | V           | Geachte mevrouw De Boer-van den Aedel |
```

Comment:
Deze persoon heeft adellijke titel GI (gravin). Dat is geen predicaat en dus valt ze niet onder regel 209/212, en mag haar adellijke gebruikt worden in de aanhef.

Reply:
Als ik het nu goed begrijp, zit er volgens mij een volgorde in rule 209/213. regel 210 is namelijk veel minder strict dan regel 213. Dus moet eerst regel 213 worden gecheckt en als aan deze regel niet wordt voldaan moet kijken of regel 210 geldt. Dit is wat ik nu in de implementatie heb gedaan om aan deze twee examples te voldoen. Klopt mijn aanname?

Reply:
>   De aanhef wordt op dezelfde manier samengesteld als voor een persoon zonder adellijke titel of predicaat in de volgende gevallen:
  - 210: Wanneer de combinatie van adellijkeTitelPredicaat en geslacht niet voorkomt in bovenstaande tabel: geslacht "O" en geen prins of prinses
  - 211: Wanneer de persoon de geslachtsnaam van de echtgenoot/partner gebruikt zonder de eigen geslachtsnaam: aanduidingNaamgebruik is "P"
  - 212: Wanneer de persoon een predicaat heeft en het geslacht is gelijk aan "V" (vrouw) en betrokkene heeft een partner
  - 213: Wanneer de persoon een predicaat heeft en het geslacht is gelijk aan "V" (vrouw) en betrokkene gebruikt de naam van de (ex)partner (aanduidingNaamgebruik is ongelijk aan "E")

Hier zit geen volgorde in. Wanneer aan één van deze condities wordt voldaan, wordt de adellijke titel of het predicaat van de persoon niet gebruikt.

Ik begrijp niet hoe je overlap of wisselwerking ziet tussen regels 210 en 213. Regel 210 heeft conditie geslacht="O". Regel 213 heeft conditie geslacht="V". Daar zit geen overlap tussen.

Er zit overigens wel overlap tussen regels 212 en 213 (beide vrouw met predicaat). 212 gaat alleen over vrouw met predicaat en een actuele partner, ongeacht het naamgebruik. 213 gaat ook over vrouw met beëindigde relatie wanneer zij de naam van de partner gebruikt. De overlap zit dan in een vrouw met predicaat en actuele partner en naamgebruik≠"E". Maar het maakt natuurlijk niet a.g.v. welke van de twee regels je beslist dat haar predicaat niet wordt gebruikt.

Reply:
Nu vallen bij mij wat kwartjes. Ik dacht dat met (ex)partner actueel of ontbonden partner wordt bedoeld, omdat er in gegeven stappen ex-partner wordt gebruikt
Zoals ik het nu interpreteer zijn regel 210-213 4 verschillende (sub-)rules. Het was voor mij niet makkelijk om te begrijpen welke van de 6 onderliggende scenario's bij welke sub-rule hoort.

Verder begrijp ik regel 210 nog steeds niet. Moet alleen bij geslacht = O de tabel worden gehanteerd?

Reply:
regel 210 zegt dat wanneer de combinatie adellijkeTitelPredicaat en geslacht niet voorkomt in de tabel vanaf regel 174, dan gebruikt je adellijkeTitelPredicaat niet. Het tweede deel van de zin (achter de dubbele punt) benoemt de concrete situaties waarbij dat geldt, namelijk geslacht='O' én adellijkeTitelPredicaat is niet "P" of "PS". 

Anders herschrijven we deze regel zo:
Wanneer de combinatie van adellijkeTitelPredicaat en geslacht niet voorkomt in bovenstaande tabel. Dit is het geval wanneer geslacht is "O" (onbekend) en adellijke titel/predicaat is ongelijk aan prins of prinses.

Maar we zouden het ook bij de eerste zin kunnen laten, als je denkt dat dat duidelijker is.

Reply:
> Ik dacht dat met (ex)partner actueel of ontbonden partner wordt bedoeld, omdat er in gegeven stappen ex-partner wordt gebruikt

Zo is het ook bedoeld:
"ex-partner" is de ontbonden partner
"(ex)partner" is de actuele of ontbonden partner

Reply:
dit is nu ook gefixt